pkg/domain/infra/abi/play.go

```diff
@@ -911,11 +912,6 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 		errors := make([]error, len(sdNotifyProxies))
 		for i := range sdNotifyProxies {
 			wg.Add(1)
-			defer func() {
-				if err := sdNotifyProxies[i].Close(); err != nil {
-					logrus.Errorf("Closing sdnotify proxy %q: %v", sdNotifyProxies[i].SocketPath(), err)
-				}
-			}()
```

Comment:
I think you also need to remove the close call below in the error case

Reply:
That's deliberate. The proxies are not returned, so in the error case, we need to close them here.

Reply:
Ah right